qiskit/transpiler/preset_passmanagers/plugin.py

```diff
@@ -301,3 +302,33 @@ def list_stage_plugins(stage_name: str) -> List[str]:
         return plugin_mgr.scheduling_plugins.names()
     else:
         raise TranspilerError(f"Invalid stage name: {stage_name}")
+
+
+def entry_point_obj(stage_name: str, plugin_name: str) -> abc.ABCMeta:
```

Comment:
I think the correct return type is `PassManagerStagePlugin`?

Reply:
Fixed in 8e8eb112e055d6cf07aa4ebf2c9208d55bc6c111 . Thanks!